atlassian/bitbucket/__init__.py

```diff
@@ -682,6 +682,23 @@ def get_repo(self, project_key, repository_slug):
         url = self._url_repo(project_key, repository_slug)
         return self.get(url)
 
+    def update_repo(self, project_key, repository_slug, **params):
+        """
+        Update a repository in a project. This operates based on slug not name which may
+        be confusing to some users.
+        :param project_key: Key of the project you wish to look in.
+        :param repository_slug: url-compatible repository identifier
+        :return: None if the repo does not exist, else the value of the put request.
+        """
+        data = self.get_repo(project_key, repository_slug)
```

Comment:
The get isn't needed becore the put because missing Keys in the put aren't changed.